client/src/app/site/base/base-sort.service/base-sort-list.service.ts

```diff
@@ -181,6 +181,14 @@ export abstract class BaseSortListService<V extends BaseViewModel>
         }
     }
 
+    private getPropertiesEqual(a: OsSortProperty<V>, b: OsSortProperty<V>): boolean {
```

Comment:
Just an idea, but have you thought about maybe making this a more generally accessible function (as in `getPropertiesEqual(a: any, b: any)` and exporting it maybe from somewhere in the infrastructure folder? This looks like the type of functionality which we might need in all kinds of different places in the future...

Reply:
Good idea, will do that

Reply:
Only extracted the array equality part since comparing sort properties is a rather special case.